experimental/dds/tree2/src/test/feature-libraries/modular-schema/modularChangeFamily.spec.ts

```diff
@@ -639,29 +587,6 @@ describe("ModularChangeFamily", () => {
 
 			assertDeltaEqual(family.intoDelta(rootChange1a), expectedDelta);
```

Comment:
The point of this test is to ensure that ModularChangeFamily appropriately calls on the intoDelta implementations for the FieldKinds that the input changeset is made of. By removing the setValue changes, but not replacing it, you've made the test check that no real delta is being generated (aside from a spine, which the ModularChangeFamily could be made to dismiss since it is empty of real changes), which could also be attained by an implementation of ModularChangeFamily that does not appropriately call on the intoDelta implementations for the FieldKinds. It would be better to replace the `setValue` with some expectation of a delete and insert (and update the `valueHandler.intoDelta` code to generate such a delta).

Reply:
Thanks for pointing this out. I missed a few updates cause tests just happened to pass.